### DISCUSSION TITLE
2024年6月緊急対応版

### DISCUSSION THREAD

**Discussion by CSReviser** [CSReviser/CaptureStream2#15] [Announcements]
2024年6月緊急対応版
＜全OS共通＞
（１）2024年6月3日(月)〜4日(火)の仕様変更に対応しました。
　　　旧JSONの更新停止　⇒　廃止　への対応。

※全ユーザーに更新適用が推奨されます。
※Mac版は動作確認していません。
※Mac版のバイナリは10.1x以前のOSに非対応です。必要な方は自分でビルドしてください。

<hr /><em>This discussion was created from the release <a href='https://github.com/CSReviser/CaptureStream2/releases/tag/20240604'>2024年6月緊急対応版</a>.</em>

**Comment by siteyuh** [comment]
MacOS（Sonoma）で動作しました。でもウインドウが少し見切れています。
<img width="652" alt="Screenshot 2024-06-05 at 0 03 52" src="https://github.com/CSReviser/CaptureStream2/assets/31228408/b2d44877-99e5-43be-8c47-7a26b72350cf">

**Reply by CSReviser** [reply]
情報提供ありがとうございます。
当方、Macがないので確認ができません。
今回の変更は該当する部分は変更していないので、現在対応方法がわかりません。
2024/04/19版以前では問題なかったのでしょうか？

**Reply by siteyuh** [reply]
以前のバージョンは全て表示されていました。
また、「カスタマイズ」はメニューバー（画面の上部）に表示されていました（Macでは一般的）。
以下以前のバージョンのウィンドウ。
![old](https://github.com/CSReviser/CaptureStream2/assets/31228408/d7c139c5-45cc-42a4-82b7-237bdab2e98b)

**Reply by CSReviser** [reply]
macOS版のメニューバーを他のOSと同じに変更したのは2023年度版からです。
2023/4版以降では今回の版が初めてということでしょうか？
Macで発生の問題についてはユーザーの方からの情報提供で手探りで解決するしかありません。
情報提供にご協力お願いします。
以下の写真はMacユーザーの方から提供頂いたものです。

![image](https://github.com/CSReviser/CaptureStream2/assets/46049273/9f162f3f-8cc3-450d-9f5d-3bfd9f46358e)

**Reply by CSReviser** [reply]
[macOS向け2024/04/13版](https://github.com/CSReviser/CaptureStream2/releases/download/20240413/CaptureStream2-MacOS-20240413.dmg)

縦方向に長くなる様に調整した版です。こちらではどの様に表示されますでしょうか？こちらの版は現在でも動作するはずです。
他のユーザーさんからの下記の写真をみて調整不要と判断しておりました。

![image](https://github.com/CSReviser/CaptureStream2/assets/46049273/87ac400d-7322-4e1e-852e-7570129e255b)

**Reply by siteyuh** [reply]
2023年2月以降は録音できていて、今回の6月に録音できなくなったわけです。
ですから、4月版は使用していませんでした。
04/13版はそちらの画像同様、見切れもなく、録音も問題なくできました。

**Reply by CSReviser** [reply]
情報提供頂きありがとうございます。
Macの環境に依存する問題で、メニューバーの高さが変わる可能性がある様に見受けられます。
対応については今後の検討課題としたいと思います。
ご協力に感謝します。

**Reply by CSReviser** [reply]
[2024/06/04縦方向調整版](https://github.com/CSReviser/CaptureStream2/releases/download/20240604/CaptureStream2-MacOS-menubar-20240604.dmg)

2024/06/04版で縦方向の調整を意図した版を作成しました。動作確認にご協力をお願いします。

**Reply by siteyuh** [reply]
ダウンロードしてみました。
前回の分より下に広がっていますが、完全には出ていませんね。
<img width="652" alt="Screenshot 2024-06-06 at 23 28 28" src="https://github.com/CSReviser/CaptureStream2/assets/31228408/eb73c17d-fff2-4afb-b0c9-872f80a45a43">


いくらでも協力させていただきます。

**Reply by CSReviser** [reply]
[2024/06/04縦方向調整版その２](https://github.com/CSReviser/CaptureStream2/releases/download/20240604/CaptureStream2-MacOS-menubar-20240604-1.dmg)

再調整版を作成しました。動作確認をお願いします。

**Reply by siteyuh** [reply]
完全に表示されるようになりました。
<img width="652" alt="Screenshot 2024-06-07 at 9 38 41" src="https://github.com/CSReviser/CaptureStream2/assets/31228408/ce7dbd42-7548-42e6-ae00-16d39f86888b">

**Reply by CSReviser** [reply]
動作確認ありがとうございます。
ノッチありMacでディスプレー表示の設定の組み合わせによって発生する事象と推測しています。期待通りの結果となりましたので、masterに取込み次回改版時に反映したいと思います。
情報提供、動作確認にご協力頂き感謝します。ありがとうございました。

**Comment by hotamon** [comment]
Mac OS Sonoma 14.5で問題なく動作しております。
ただ，以前は，すでにダウンロードしてあるものは，スキップする仕様でしたが，
最近の版は，常にダウンロードして上書きする仕様になっています。
以前のように，スキップする仕様にするのは難しいでしょうか？

**Reply by CSReviser** [reply]
「既存のファイルはスキップ」のボタンにチェックがついている状態で機能していないという事でよろしいでしょうか？
この機能は変更していず、当方の環境では不具合現象を確認できていません。
当方はMacの実機を持っていないのでMac固有の問題の可能性があります。
詳細な情報提供をお願いします。
・正常に動作したバージョン
・不具合が発生したバージョン
・不具合を確認した番組
・不具合が発生した前後でMac環境の変更がなかったか？
・その他、不具合の発生前後で何か変更など、手がかりとなりそうな情報

また、一度設定を削除して再起動をお試しください。
「カスタマイズ」から「設定削除（終了）」を選択、メッセージ窓が開いたら「Yes」をクリック。CS2が終了するので再度起動して、動作を確認して下さい。

**Reply by CSReviser** [reply]
Macユーザーの方から「既存のファイルはスキップ」は機能しているとの情報を頂きました。個別の環境で発生する問題や特定の条件でのみ発生する問題の可能性が考えられます。
まずは設定削除をお試しください。

＜動作報告頂いた環境＞
iMac 2019（Intel Core i5）
CaptureStream2-MacOS-menubar-20240604-1.dmg

**Reply by hotamon** [reply]
ご返答ありがとうございました。
ご指摘のとおり，設定解除後，再起動したら，スキップしてくれました。
私の固有の問題だったようです。
ご迷惑をおかけいたしました。
